README.md

```diff
@@ -1,49 +1,64 @@
 
 This repository contains the official documentation for the eXist-db Native XML database and the application for browsing it.
 
 ## Resources
 
-- Browse the latest release of the documentation at http://exist-db.org/exist/apps/doc/. 
-- The documentation app is included by default in the eXist-db installer. Just go to your eXist server's Dashboard and select Documentation.
-- Update to the latest release via the eXist-db package manager or via the eXist-db.org public app repository at <http://exist-db.org/exist/apps/public-repo/>.
+-   Browse the latest release of the documentation at [http://exist-db.org/exist/apps/doc/](http://exist-db.org/exist/apps/doc/).
+-   The documentation app is included by default in the eXist-db installer. Just go to your eXist server's Dashboard and select Documentation.
+-   Update to the latest release via the eXist-db package manager or via the eXist-db.org public app repository at <http://exist-db.org/exist/apps/public-repo/>.
 
 ## Building from source
 
-1. Dependencies: Maven 3.x
-
-2. Clone the repository to your system:
+1.  Dependencies: Maven 3.x
 
+2.  Clone the repository to your system:
 ```bash
 $ git clone https://github.com/exist-db/documentation.git exist-documentation
 ```
 
-3. Build the documentation application:
+3.  Build the documentation application:
 ```bash
 $ cd exist-documentation
 $ mvn clean package
 ```
 
-4. An EXPath Application Package (.xar file) is deposited in the `target` directory
+4.  An EXPath Application Package (.xar file) is deposited in the `target` directory
+
+5.  Install this file via the Dashboard > Package Manager.
 
-5. Install this file via the Dashboard > Package Manager.
+Find an area of the documentation that needs to be improved? Please raise an [issue](https://github.com/eXist-db/documentation/issues) or submit a [pull request](https://github.com/eXist-db/documentation/pulls)!
 
-Find an area of the documentation that needs to be improved? Please raise an issue and submit a pull request!
+Our test-suite performs a validation check on all articles in the `/data` directory when you open a pull request. You can speed up the review process by validating your edits locally before submitting a pull request.
 
 
 ## Building a Release from source
 
-1. Follow the instructions from [Building from source](#building-from-source)
+1.  Merge outstanding and reviewed PRs.
+
+2.  Set version numbers in `pom.xml`.
+  2. Keep in mind, that the major version of the documentation app mirrors the [major version](https://github.com/eXist-db/exist/blob/develop/exist-versioning-release.md#versioning-scheme) the current eXist release. So for exist-db v3.x.x the documentation app will be v3.y.y. Minor and Patch numbers can vary between the two.
+  3. You should also check and set the appropriate `exist.version` to prevent users of older exist releases, from installing the wrong documentation locally.
```

Comment:
Actually I don't see any processor dependency in `xar-assembly.xml` is something missing?

Reply:
Oh I thought that the exist.version in pom would insert that automatically, should contributors edits this manually in xar-assembly or can we automate that in the release process?